### ISSUE TITLE
Finish plot_island_tbl on tree_plots branch

### ISSUE THREAD

**Issue by richelbilderbeek** [joshwlambert/DAISIEmainland#51]
Just make the tests in `test-plot_island_tbl.R` produce beautiful tables.

No need to separate between ideal and empircal data yet, this depends on #55.

**Comment by richelbilderbeek** [comment]
Current state:

![Screenshot from 2022-01-11 06-13-12](https://user-images.githubusercontent.com/2098230/148885177-9b869f0e-0bae-41db-8f52-5f2a69106d5e.png)

**Comment by richelbilderbeek** [comment]
Some info from Josh:

 * The col_t given to each cladogenetic species (spec_type == “C”) is the same, it is the time clade colonised
* The branch_t of the first cladogenetic species in the clade is the same as the col_t, the branch_t of the other species in the clade is the time they branched
* The branch_code is different from mainland_clade, species that don’t speciate have branch_code == NA. 
* A cladogenetic speciation event causes one new row to be added, unlike mainland_clade where two rows are added

**Comment by richelbilderbeek** [comment]
@joshwlambert is this picture correct? I'd enjoy to be corrected here :-)

![current_state_annotated](https://user-images.githubusercontent.com/2098230/148886099-b569a1ed-0af3-4f32-bb39-e281e0de78c4.png)

**Comment by joshwlambert** [comment]
@richelbilderbeek the figure is correct. The two green lines should form the clade you have drawn on the left-hand side.

**Comment by richelbilderbeek** [comment]
@joshwlambert great that the figure is correct, with some luck I will finish it today.

AFAICS, I never get info on clades that went extinct before the present. Am I correct [1]? 

 * [1] Whether they should be in is another question altogether. Let's talk after the next release :-)

**Comment by joshwlambert** [comment]
@richelbilderbeek the island_tbl only contains reconstructed information so no extinct species are kept.

**Comment by richelbilderbeek** [comment]
For now, this Issue is on hold until after the next release. I will close it for now.
